<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="WelcomePg">
     <widget class="QLabel" name="Welcome">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CustomMeas">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>200</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Custom Measurements</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LabFiles">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>290</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Laboratory Files</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ParamMeas">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>380</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Parameter Measurements</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SavedVals">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>470</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Saved Values</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>680</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QFrame" name="VoltageMod">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>241</width>
        <height>481</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Voltage</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="V1">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="V2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="V3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>290</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="V4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>390</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>V1 Meas - </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>V2 Meas - </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>V3 Meas - </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>390</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>V4 Meas - </string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="CurrentMod">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>241</width>
        <height>481</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Current</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="I1">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="I2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="I3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>290</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLCDNumber" name="I4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>380</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>I1 Meas - </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>I2 Meas - </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>I3 Meas - </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>380</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>I4 Meas - </string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="SpeedMod">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>110</y>
        <width>241</width>
        <height>481</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="BasicIOMod">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>110</y>
        <width>241</width>
        <height>481</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Basic I/O</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QPushButton" name="ReturnHome">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>670</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>170</y>
        <width>251</width>
        <height>451</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>232</width>
         <height>329</height>
        </rect>
       </property>
       <widget class="QCheckBox" name="Task1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777183</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Task 1</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
       <widget class="QCheckBox" name="Task2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Task 2</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="Task3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Task 3</string>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QCheckBox" name="Task4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Task 4</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="Task5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Task 5</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="TaskList">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>80</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Lab Tasks</string>
      </property>
     </widget>
     <widget class="QLabel" name="LabName">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>26</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Laboratory #1</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="LabMeasView">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>621</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(64, 65, 66);</string>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>481</width>
          <height>331</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>LabCircuit.png</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Circuit View</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Measurement List</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Graph View</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Table View</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>181</width>
        <height>471</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="MeasView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Measurement View - </string>
       </property>
      </widget>
      <widget class="QPushButton" name="CircuitView">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Circuit View</string>
       </property>
      </widget>
      <widget class="QPushButton" name="MeasListView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>131</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Measurement List</string>
       </property>
      </widget>
      <widget class="QPushButton" name="GraphView">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Graph View</string>
       </property>
      </widget>
      <widget class="QPushButton" name="TableView">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Table View</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
